### ISSUE TITLE
Make sure bot does not automerge PRs that are not created by another bot

### ISSUE THREAD

**Issue by derberg** [asyncapi/.github#78]
Here bot accepted and automerge was triggered https://github.com/asyncapi/spec-json-schemas/pull/104

job log https://github.com/asyncapi/spec-json-schemas/runs/3732367755

**Comment by KhudaDad414** [comment]
@derberg is this bug still valid?

**Comment by derberg** [comment]
recreating will be difficult, you simply need to understand the flow:

- autoapprove approves anything that comes from specific actor
- automerge merges anything that comes from specific actor

what happened there:

- during work on release, the release branch was updated by bot with a package json bump, but the release branch was not yet supposed to be merged

my assumption is that the problem is that we verify the `actor` of the PR event, instead of the PR submitter. It should be enough to change the autoapprove and automerge to work only if `PR owner is bot` and not that PR bot pushed a commit.

makes sense?
so basically you just need to figure out where in the payload you have such info 🤔 
might be that I need something like this https://github.com/asyncapi/.github/pull/94/files#diff-19a2727046458ad005aac55037c42104974c21565d593fbdbaa35724d9a8447cR26 too 🤔
